README.md

```diff
@@ -1 +1,68 @@
-# train-react-insight
\ No newline at end of file
+<h1>Budgeting Application</h1>
```

Comment:
README files use the markdown syntax so probably a better idea to structure this in Markdown rather than HTML